lib/utils/testem-events.js

```diff
@@ -152,31 +161,56 @@ class TestemEvents {
     };
   }
 
+  /**
+   * Creates a file with targetJsonObject
+   *
+   * @param {*} fileName
+   * @param {*} targetJsonObject
+   * @param {*} option
+   */
+  _writeJsonToFile(fileName, targetJsonObject, option = {}) {
+    try {
+      fs.writeJsonSync(fileName, targetJsonObject, option);
+    } catch (err) {
+      err.file =  err.file || fileName;
```

Comment:
the value provided to the catch clause in JavaScript is funky, it can literally be anything. To avoid our catch clause itself crashing, I recommend that we are more defensive here, and ensure that the error is in-fact an object who can reasonably have properties set on it.

```
if (typeof err === 'object' && err !== null ) {
    err.file = err.file || fileName;
}

throw err;
```